src/pages/Items.jsx

```diff
@@ -0,0 +1,74 @@
+import { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import ItemList from '../components/ItemList';
+import Dropdown from '../components/Dropdown';
+import logoBig from '../assets/logo.png';
+import logoSmall from '../assets/logo_text.png';
+import icProfile from '../assets/ic_profile.svg';
+import icSearch from '../assets/ic_search.svg';
+import './Items.css';
+import Pagenation from '../components/Pagenation';
+
+function Items() {
+  document.title = "중고마켓";
+
+  const [mode, setMode] = useState(window.innerWidth < 768 ? "Mobile" : (window.innerWidth < 1200 ? "Tablet" : "PC"));
+  const [order, setOrder] = useState("recent");
+  const [totalCount, setTotalCount] = useState(1);
+  const [page, setPage] = useState(1);
+
+  const handleResize = () => {
+    if (window.innerWidth < 768) { setMode("Mobile"); }
+    else if (window.innerWidth < 1200) { setMode("Tablet"); }
+    else { setMode("PC"); }
+  }
+
+  window.onresize = handleResize;
```

Comment:
💊 제안
컴포넌트에 `window.onresize = handleResize;`를 직접 두면 매 렌더마다 핸들러가 재할당되어 불필요한 연산이 발생하고, 컴포넌트가 언마운트된 후에도 이전에 할당된 핸들러가 해제되지 않아 예기치 않은 동작을 초래할 수 있습니다~
아래처럼 useEffect 안에서 이벤트를 바인딩하고 해제하는 패턴을 사용하세요!

```suggestion
useEffect(() => {
  window.addEventListener('resize', handleResize);

  return () => {
    window.removeEventListener('resize', handleResize);
  };
}, []);
```